src/Propulsion.SqlStreamStore/SqlStreamStoreSource.fs

```diff
@@ -32,12 +32,12 @@ type SqlStreamStoreSource
         // The whitelist of Categories to use
         ?categories,
         // Predicate to filter Categories to use
```

Comment:
it's always the comments! (I'm getting closer to driving myself to implement the `///` comments on arguments in the compiler!)

Reply:
Looks great now, thanks - will merge when releasing (or ping me if you just want it on master already)